bootstrap.sh

```diff
@@ -305,6 +307,8 @@ function set_VENDOR {
         AUTORECONF_REBUILD_HOST=true
       elif [[ "x$VENDOR_RELEASE" == 'xquantal' ]]; then
         AUTORECONF_REBUILD_HOST=true
+      elif [[ "x$VENDOR_RELEASE" == 'xtrusty' ]]; then
```

Comment:
Is it expected that `raring` doesn't appear between `quantal` & `trsusty`?
Because it does appear in lines 229-230 above.